vroom/ruff.vroom

```diff
@@ -21,15 +21,15 @@ system.
   :silent file somefile.py
   % f()
   :FormatCode ruff
-  ! ruff format --stdin-filename=somefile.py.*
+  ! ruff format - --stdin-filename=somefile.py.*
```

Comment:
Is there a reason to expect something at the end of the line?

Reply:
Yes, the syscall lines from vroom always have some `2>some_outfile` stuff at the end.